src/datalad_installer.py

```diff
@@ -1287,6 +1287,7 @@ def install_package(
         log.info("Extra args: %s", extra_args)
         if kwargs:
             log.warning("Ignoring extra installer arguments: %r", kwargs)
+        runcmd("brew", "update")
```

Comment:
Thanks!
I will merge as is so we could test it right away in datalad CI, but please submit a follow up PR so it would use some class attribute to annotate if it ran update already so it would not do it for the next  `install` since IMHO then there should be no need